pyscraper/sp/findquotation.py

```diff
@@ -81,10 +81,10 @@ def find_id_for_quotation(self,date_string,regexp_list,column_numbers_strings):
                 if k not in found_ids_with_correct_column:
                     del self.ids_with_quote[k]
         for k in self.ids_with_quote:
-            occurences = self.ids_with_quote[k]
-            if occurences > max_occurences:
+            occurrences = self.ids_with_quote[k]
+            if occurrences > max_occurences:
```

Comment:
Any reason you didn't correct `max_occurrences` also? :)

Reply:
I found the spelling mistakes using codespell, then checked the results and removed a few false positives. I didn't spot that codespell had missed max_occurences.